hermes-console/static/partials/modal/editSubscription.html

```diff
@@ -170,6 +170,64 @@ <h3 class="modal-title" ng-show="operation === 'EDIT'"><small>Edit:</small> {{su
                 </div>
             </div>
 
+            <hr />
```

Comment:
I did a little bit of hacking in my browser to make the layout more concise and clean, in general I got rid of some bootstrap borders/margins. Please, have a look. I'll add comments regarding specific changes. :)

![screen shot 2018-10-05 at 11 34 36](https://user-images.githubusercontent.com/2399036/46527969-dd658b00-c892-11e8-9ec4-f1618f9ae5f4.png)